packages/bundle/test/dimensions.test.js

```diff
@@ -59,7 +59,7 @@ describe('dimensions', function () {
         assume(svg).equals(item);
 
         assume(item.viewBox).is.a('string');
-        assume(item.viewBox).equals('16.072994232177734 55.62681579589844 493.87457275390625 509.17413330078125');
+        assume(item.viewBox).equals('0 0 0 0');
```

Comment:
This seems really weird, and no way this is valid. It's getting height and width of 0.

Reply:
how is this related to what I'm doing though? I'll investigate..

Reply:
fixed with latest commit, puppeteer hack is no longer need due to upstream fixes https://github.com/GoogleChrome/puppeteer/issues/728#issuecomment-352005352